test/publishing_api_v2_test.rb

```diff
@@ -511,6 +511,152 @@ def content_item_for_content_id(content_id, attrs = {})
     end
   end
 
+  describe "#republish" do
+    describe "if the republish command succeeds" do
+      before do
+        publishing_api
+          .given("a unpublished document exists with content_id: #{@content_id}")
+          .upon_receiving("a republish request")
+          .with(
+            method: :post,
+            path: "/v2/content/#{@content_id}/republish",
+            headers: GdsApi::JsonClient.default_request_with_json_body_headers.merge(
+              "Authorization" => "Bearer #{@bearer_token}"
+            ),
+          )
+          .will_respond_with(
+            status: 200
+          )
+      end
+
+      it "responds with 200 if the publish command succeeds" do
+        response = @api_client.republish(@content_id)
+        assert_equal 200, response.code
+      end
+    end
+
+    describe "if the content item does not exist" do
+      before do
+        publishing_api
+          .given("no content exists")
+          .upon_receiving("a republish request")
+          .with(
+            method: :post,
+            path: "/v2/content/#{@content_id}/republish",
+            headers: GdsApi::JsonClient.default_request_with_json_body_headers.merge(
+              "Authorization" => "Bearer #{@bearer_token}"
+            ),
+          )
+          .will_respond_with(
+            status: 404
+          )
+      end
+
+      it "responds with 404" do
+        error = assert_raises GdsApi::HTTPClientError do
+          @api_client.republish(@content_id)
+        end
+
+        assert_equal 404, error.code
+      end
+    end
+
+    describe "if the update information contains a locale" do
+      before do
+        publishing_api
+          .given("a draft content item exists with content_id: #{@content_id} and locale: fr")
```

Comment:
This one doesn't seem right as we can't do this on a draft. If we want to do this with a pact test we'll need to add another set-up clause. Though I don't really think this adds a huge amount of value so I'd suggest we just remove this describe since setting up for it requires us to do additional work.

Reply:
sure